docs/new-editor.md

```diff
@@ -0,0 +1,163 @@
+# Integrating a new editor
+
+Before writing a new editor client, first check if someone else has managed to
+integrate metals with your favorite text editor.
+
+* [VS Code](https://github.com/scalameta/metals/blob/master/vscode-extension/src/extension.ts),
+  maintained in this repo
+* [Atom](https://github.com/laughedelic/atom-ide-scala), maintained by
+  [@laughedelic](https://github.com/laughedelic)
+* [Emacs](https://github.com/rossabaker/lsp-scala), maintained by
+  [@rossabaker](https://github.com/rossabaker)
+* Others, see [#217](https://github.com/scalameta/metals/issues/217). Please
+  open an issue or ask on [gitter](https://gitter.im/scalameta/scalameta) if you
```

Comment:
Did you ever manage to create a dedicated gitter channel `scalameta/metals`? I find the `scalameta/scalameta` channel a bit noisy to follow some days.

Reply:
I finally managed to create a channel! 🎉 it wasn't easy, had to delete my gitter account, revoke scalameta org access, and redo everything from scratch + fiddle a bit with admin settings in scalameta. Behold https://gitter.im/scalameta/metals

Reply:
seriously. 😮